landcover.mss

```diff
@@ -340,13 +341,18 @@
   [feature = 'natural_grassland'][zoom >= 8],
   [feature = 'landuse_meadow'][zoom >= 8],
   [feature = 'landuse_grass'][zoom >= 10],
+  [feature = 'landcover_grass']
   [feature = 'landuse_village_green'][zoom >= 10],
-  [feature = 'leisure_common'][zoom >= 10] {
+  [feature = 'leisure_common'][zoom >= 10]
```

Comment:
Could you keep the syntax as the rest of the file for consistency ?
```
[feature = 'leisure_common'][zoom >= 10] {
```

Reply:
Fixed